subjects/mobile-dev/chess/README.md

```diff
@@ -12,12 +10,35 @@ So whether you're a seasoned grandmaster or a newcomer to the game, join the ran
 
-Your task is to develop a mobile app that allows users to play chess with each other. This is a fullstack app so you will need to implement both backend and frontend. You may use any desired backend technology as long as you're following the [backend routes.](https://github.com/alem-01/chess). Good luck!
+### Instructions
+
+Your task is to develop a mobile app that allows users to play chess with each other. This is a fullstack app so you will need to implement both backend and frontend. You may use any desired backend technology as long as you're following the [backend routes.](https://github.com/alem-01/chess).
+
+Make sure that your app has the following requirements:
+
+- Must have UI screens, including at least the following:
+  - A Main menu
+  - A Game board
+  - A waiting room
+- Must implement the logic to validate moves and enforce the rules of chess, disallowing illegal moves.
+- Must ensure that white pieces always play first in each game.
+- Must have a multiplayer mode which enables players to challenge and play against other users in real-time matches.
+- Must have the ability for players to invite each other to a game or join a public game.
+- Must have a waiting room where players can wait for opponents and initiate game sessions.
+- Must have a notification system to alert players when it is their turn to make a move.
+
+### Hints
 
-Here are some additional considerations to keep in mind as you work on this exercise:
+- Configure your backend solution for managing the multiplayer functionality and game data storage.
+- Create the initial UI screens, including the main menu, waiting room and game board.
+- Implement the logic for validating and enforcing the rules of chess, disallowing illegal moves, and determining the turn order.
+- Research different chess rules and app designs to get ideas for your own app.
+- Think about how you will handle game state and moves within the app, and how you will ensure that the game is played fairly and accurately.
 
-1. Research different chess rules and app designs to get ideas for your own app.
-2. Think about how you will handle game state and moves within the app, and how you will ensure that the game is played fairly and accurately.
-3. Implement backend according to the swagger:
+### Bonus
 
-By taking these factors into account, you can create a successful and engaging chess app that meets the objectives of the exercise.
+- **Game Over**, when the game reaches a terminal state (checkmate, stalemate, or draw), we can send a message to both players indicating that the game is over.
```

Comment:
This point and the below ones seems exactly the same ask to me (same thing as the open comment in the audit file)